packages/squiggle-lang/src/rescript/Reducer/ReadMe.md

```diff
@@ -0,0 +1,15 @@
+To interface your library there only 2 files to be modified:
+- Reducer/Reducer_Extension/Reducer_Extension_CodeTreeValue.res
+
+  This is where your additional types are referred for the dispatcher.
+
+- Reducer/Reducer_Extension/Reducer_ReducerLibrary.res
```

Comment:
Also--- would you like to leverage interfaces to enforce that the `reducer` internals are not messed with? Ozzie's comment about [interfaces](https://rescript-lang.org/docs/reason-compiler/latest/automatic-interface-generation) is somewhere around here

Reply:
Ok. I will check the interfaces... But I don't think so far I have exposed something that should not be... Also remember that for something to be unit tested, that something has to be exposed...

Reply:
(One post about this topic, that lightly mentions this discussion: https://dev.to/yawaramin/ocaml-interface-files-hero-or-menace-2cib)

Reply:
If you ask me then I want to forbid everything but Reducer.res. All the rest is inner work needing each other. I am not supplying a number of useful modules under Reducer name space... If it was a single mega file then I would be writing an interface banning all those modules alltogether